cmd/traffic_layout/traffic_layout.cc

```diff
@@ -172,6 +172,7 @@ produce_layout(bool json)
   print_var("remap.config", RecConfigReadConfigPath("proxy.config.url_remap.filename"), json);
   print_var("plugin.config", RecConfigReadConfigPath(nullptr, "plugin.config"), json);
   print_var("ssl_multicert.config", RecConfigReadConfigPath("proxy.config.ssl.server.multicert.filename"), json);
+  print_var("ssl_server_name.config", RecConfigReadConfigPath("proxy.config.ssl.servername.filename"), json);
```

Comment:
Why isn't this `SSL_SERVER_NAME_CONFIG`?

Reply:
I was matching with the other print_vars -_-